### PR TITLE
#170602091 User should be able to make one way trip request 

### PR DESCRIPTION
**Pull request by william000000**
#### What does this PR do?
User should be able to make oneway trip request
#### Description of Task to be completed?
- Design Oneway Trip Page
- Consume its API
- Tests
- Refactoring oneway and round trip
- Add more reusable components( Select City Fields, Header/Title of Trip Request and Left Side Button)
#### How should this be manually tested?
- Clone this repository using the command: `https://github.com/andela/team-odd-bn-frontend.git`
- Checkout to this branch using the command: `git checkout ft-one-way-trip-request-170602091`
- Run the application using the command: `npm run dev` to test it in the browser

#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
[#170602091](https://www.pivotaltracker.com/story/show/170602091)
#### Screenshots (if appropriate)
image
image

#### Questions:
- N/A

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -23,9 +23,13 @@ import Profile from './components/Profile';
...
 
-dotenv.config();
+import Dashboard from './views/Dashboard/sidebar/index';
+import OneWayTrip from './containers/trips/oneway';
 
-axios.defaults.BASE_URL = process.env.BASE_URL;
+import './assets/css/style.scss';
+import './assets/css/App.scss';
+
+axios.defaults.baseURL = 'https://team-odd-bn-backend-staging.herokuapp.com/api/v1/users';
```
Comment (hezronkimutai) [inline src/App.js]: Hi @willy, put this in the .env

File: src/__tests__/actionTests/profileActionTests.js [hunk 2]
```diff
@@ -135,48 +135,48 @@ describe('Profile Actions Test Suite', () => {
...
+  //   await store.dispatch(uploadImage())
+  //     .then(async () => {
+  //       const calledActions = store.getActions();
+  //       expect(calledActions).toEqual(expectedActions);
+  //     });
+  // // });
+  // it('Should dispatch UPLOAD_IMAGE_ERROR', async () => {
+  //   moxios.wait(() => {
+  //     const request = moxios.requests.mostRecent();
+  //     request.respondWith({
```
Comment (hezronkimutai) [inline src/__tests__/actionTests/profileActionTests.js]: Don't forget to remove these comments.

File: src/assets/css/App.scss [hunk 3]
```diff
@@ -12,3 +12,26 @@ $logo-color: #00B9F2;
...
+  .btn-short {
+    width: 60%;
+  }
+
+  button:hover {
+    background-color: white;
+    color: #00b9f2;
+    border: solid 1px #00b9f2;
+  }
+}
```
Comment (hezronkimutai) [inline src/assets/css/App.scss]: eol

File: src/assets/css/base.scss [hunk 4]
```diff
@@ -157,29 +163,109 @@
...
     }
+
   }
 }
+
+
 ::-webkit-scrollbar {
   width: 0px;
-}
+}
```
Comment (hezronkimutai) [inline src/assets/css/base.scss]: add eol.

**Review by hezronkimutai — COMMENTED** [review]
Hi @william000000 , Left some comments,.

**Review by NiyongaboEric — CHANGES_REQUESTED** [review]
Good job @william000000. Also, ensure the following:
- Fixing test
- Ensure one way trip request works on Heroku
- Trips sidebar should not overflow the top navbar

Screen Shot 2020-01-17 at 10 09 41

**Review by hezronkimutai — APPROVED** [review]
Nice job.

**Review by NiyongaboEric — APPROVED** [review]

**Review by nziokaivy — APPROVED** [review]
Great job! @Willy

**Review by shemaeric — APPROVED** [review]
Good work @william000000

**Pull request merged**